pyproject.toml

```diff
@@ -1,261 +1,266 @@
-[build-system]
-requires = ["setuptools>=61.2", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools.packages]
-find = {namespaces = false}
-
-[tool.setuptools.package-dir]
-"" = "."
+[tool.poetry]
+# required ones
+name = "geoips"         # geoips package available at the root of the project
+version = "0.0.0"
+description = "Geolocated Information Processing System"
+authors = ["Tom <tom.jerry@colostate.edu>"]
 
-[tool.setuptools.package-data]
-"*" = [
-    "plugins/*.txt",
-    "plugins/*/*.txt",
-    "plugins/*/*/*.txt",
-    "plugins/*/*/*/*.txt",
-    "plugins/*/*/*/*/*.txt",
-    "plugins/*.yaml",
-    "plugins/*/*.yaml",
-    "plugins/*/*/*.yaml",
-    "plugins/*/*/*/*.yaml",
-    "plugins/*/*/*/*/*.yaml",
-    "schema/*.yaml",
-    "schema/*/*.yaml",
-    "schema/*/*/*.yaml",
-    "schema/*/*/*/*.yaml",
-    "schema/*/*/*/*/*.yaml",
+# optional ones
+maintainers = ["Tom <tom.jerry@colostate.edu>"]
+#homepage = "set the url"   // optional
+repository = "https://github.com/NRLMMD-GEOIPS/geoips"    #optional
+#documentation = "set the url"      // optional
+#license = {file = "LICENSE"}   // required
+readme = "README.md"
+keywords = ["GeoIPS", "Poetry"]
+packages = [{include = "geoips"}]  # not needed as tool.poetry.name is properly set already
+classifiers = [                # // list of PyPI trove classifiers to describe the project
+    "Programming Language :: Python :: 3", ]
+include = [
+    "*.txt",
+    "*.py",
+    "*.yaml",
+    "yaml_configs/*",
```

Comment:
Does the include now include all levels ? (ie, can we get rid of the "yaml_configs" entries here)?  Or do we need to add back in all of the plugins/*/*/* and schema/*/*/* lines? @jsolbrig

Reply:
I'm looking into this now. It looks like this is one spot where Poetry may be nicer than Setuptools. It looks like this will work:
```toml
include = [
    "**/*.py",
    "**/*.yaml",
    "**/*.txt"
]
```